components/source/default/index.mjs

```diff
@@ -1,12 +1,10 @@
-/* globals URL */
-
 const { parse: parseJSON } = JSON;
 const _URL = URL;
```

Comment:
This is to protect against modifications of the global object by the app being recorded. Node does this as well. (It even goes further by using functions applications rather than method invocations because prototypes can be altered as well -- cf: https://github.com/nodejs/node/blob/12e3c74e2edf78eeae662dce0a9db17a6fe7d730/lib/internal/per_context/primordials.js). I'm considering using an eslint rule to forbid global variables so that this pattern can be consistently enforced. Now, I'm sure there are places where I forgot to use this pattern because it is not detected by the test suite.

Reply:
Thanks for this.